WordPress/Classes/Extensions/UIImageView+Networking.swift

```diff
@@ -22,9 +29,16 @@ extension UIImageView
             placeholderImage: placeholderImage,
             success: { [weak self]
                 (request: NSURLRequest, response: NSHTTPURLResponse?, image: UIImage) -> Void in
-                
-                self?.image = image
-                success?(image)
+
+                let processedImage: UIImage
```

Comment:
Yeah, I went for the longer version. I didn't wanted to introduce much change before the refactor but my initial plan was for the `processor` value to not be optional and have a default value of `identity`:

```swift
func identity<T>(x: T) -> T {
    return x
}
```

I still have to think more about if that's really better than the optional version.